src/sponsors/exhibitor-guide.njk

```diff
@@ -0,0 +1,363 @@
+---
+title: Exhibitor's Guide
+description: PyCon India 2025 Exhibitor's Guide
+layout: base.njk
+---
+
+{% from "components/button.njk" import button %}
+{% from "components/flat-card.njk" import flatCard %}
```

Comment:
please remove flatCard since it is not being used

Reply:
Please have a look at https://github.com/pythonindia/inpycon2025/blob/main/src/cfp/index.njk how listCards are used, it will help you reduce a lot of code.

Reply:
You can either use listCard or list component